### DISCUSSION TITLE
Possibility for USB Boot Support for SSD

### DISCUSSION THREAD

**Discussion by themajesticking** [tesla-android/issue-tracker#164] [Ideas]
I have several SSDs laying around that I use for various projects and considering the Pi4 works great using a SSD as a boot drive I was wondering if USB boot is something that could be supported in the future. Last time I checked I think the dev was using Uboot. Does UBoot have USB boot support?

**Comment by AlexNoop** [comment]
Hey ! same problem ;) i use Argon One M2 case, and no boot

**Comment by mikegapinski** [comment]
Usb boot support is not there and is not planned for now

**Reply by AlexNoop** [reply]
Not bad ;) 
Just use SD for now ! Thanks 😍
